test/unit/util/conf/test_master_config_loader.py

```diff
@@ -0,0 +1,46 @@
+from app.util.conf.master_config_loader import MasterConfigLoader
+from app.util.conf.configuration import Configuration
+from test.framework.base_unit_test_case import BaseUnitTestCase
+
+
+class TestMasterConfigLoader(BaseUnitTestCase):
+
+    def test_configure_default_sets_protocol_scheme_to_http(self):
+        mock_config_file = self.patch('app.util.conf.base_config_loader.ConfigFile').return_value
+
+        config = Configuration.singleton()
+        config_loader = MasterConfigLoader()
+        config_loader.configure_defaults(config)
+
+        key = 'protocol_scheme'
+        expected_stored_protocol_scheme_value = 'http'
+        actual_stored_protocol_scheme_value = Configuration[key]
+
+        self.assertEqual(expected_stored_protocol_scheme_value, actual_stored_protocol_scheme_value,
+                         'The configuration value for the key "{}" was expected to be {}:{}, but was {}:{}.'.format(
```

Comment:
I'm not sure how much value we get out of providing a custom error message. Does something bad happen if the assert fails and this message is not provided?